src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -90,10 +90,21 @@ def arch_repo_table_cell(repo, arch, package_name)
                         { title: link_title, rel: 'nofollow' }
                        )
     end
-    result += '</td>'.html_safe
+
+    status_desc = get_package_status_description(status['code'])
+
+    result += " <img title='#{status_desc}' class='icons-help' alt='#{status_desc}' src='/assets/s.gif' /></td>".html_safe
     result
   end
 
+  def get_package_status_description(status)
+    if PACKAGE_STATUS_LIST.has_key?(status)
+      return PACKAGE_STATUS_LIST[status]
+    end
+
+    "status explanation not found"
```

Comment:
It can be more simple I guess:
```ruby
def get_package_status_description(status)
  PACKAGE_STATUS_LIST[status] || "status explanation not found"
end
```